cfme/tests/cloud/test_cloud_proxy.py

```diff
@@ -0,0 +1,149 @@
+# -*- coding: utf-8 -*-
+# These tests don't work at the moment, due to the security_groups multi select not working
```

Comment:
Is this comment still supposed to be here? Let's put it somewhere more functional if so. If that's the case I'd expect PRT failures but don't see any.

Is this a GH issue? if so add it as a blocker so the tests don't run until we fix the framework.